site/content/kapp-controller/docs/latest/faq.md

```diff
@@ -38,3 +38,59 @@ Even though the error above is reported, the Package will still be installed and
 should work as expected. It can also still be uninstalled by deleting the
 PackageInstall. The PackageRepository can be recreated and the PackageInstall
 will sync and reconcile without any updates needed to resolve the error.
+
+### How can I generate the valuesSchema from my ytt schema?
+
+If you are using `ytt` as your Package's templating option and have [defined a schema](../../../ytt/docs/latest/how-to-write-schema), you can use `ytt` to generate your `valuesSchema` (which is in OpenAPI v3 format) for you.
+
+This is the recommended workflow:
+
+1. Create an OpenAPI Document from a Data Values Schema file:
+
+    ```bash
+    $ ytt -f schema.yml --data-values-schema-inspect -o openapi-v3 >schema-openapi.yml
+    ```
+   
+    which will produce...
+
+    ```yaml
+    #! schema-openapi.yml
+    openapi: 3.0.3
+    info:
+      version: 1.0.0
+      title: Openapi schema generated from ytt schema
+    paths: {}
+    components:
+      schemas:
+        dataValues:
+          type: object
+          properties:
+            namespace:
+              type: string
+              default: fluent-bit
+    ```
+
+2. Turn your Package CR into a `ytt` template, itself:
+
+    ```yaml
+    #! package.tpl.yml
+    #@ load("@ytt:data", "data")
+    #@ load("@ytt:yaml", "yaml")
+    #! package.yml
```

Comment:
```suggestion

```
The filename is already on line 76, and `package.yml` is the output file name, so we don't need it here I believe.